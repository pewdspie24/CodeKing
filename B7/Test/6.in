20
24 971310339
3051440 1207465
6151747 6544898
2146917 7153423
7657567 7776252
4213081 3366257
696237 4740171
3014285 4969207
937424 4254741
2617895 9524668
992685 6556149
4748084 8588789
8713229 9136688
6299432 9365573
2979424 2290632
780175 7948298
9475864 6046160
3109455 3565039
524759 6199158
6921200 3219142
2868425 5076877
2563574 7100533
3349996 4711087
5239870 8836621
6721784 4457935
12 433528291
937778 1972150
6732552 7547916
6815386 8138089
9384321 6587036
6387655 6179265
5976425 200707
8154395 1078195
5631889 2869824
5759237 5473971
5702214 5710087
5429808 2368790
9677148 4954619
25 848549970
9285251 3977781
6940557 6481735
7787887 7520697
5247702 3664640
3472842 5492796
9545442 1743675
5584353 4485775
6379927 4863902
2770366 8833034
6910455 2202523
6250587 2189935
6248183 203023
8009323 6996827
6746252 4149602
5888864 4509882
535820 7183133
9520774 503273
8577061 2310555
5571101 2904295
9569735 5917832
6290544 2413233
2333521 6735466
1034174 8671021
8906111 8334405
6634276 5744970
12 175742045
8695766 3930145
8035419 5484941
6056719 8179838
7527015 4697522
8656072 9933817
8847450 2838960
1764212 2297543
1902072 9856156
4184648 3913238
8484886 4118054
240334 9863300
6946564 3056947
25 106755610
5154914 4095758
8576354 7213749
6249023 127120
8343748 8552473
7675050 1788231
9442298 236393
6377854 2373040
8352087 7257570
7255995 394606
203959 7058419
2917484 8638151
9641015 1731774
7578853 9377461
2491185 7540891
8690796 1590505
4807877 9486304
5864667 489343
744730 3210157
488815 9971637
843121 2935889
489528 4403355
4204170 3235663
4184073 4418931
4980049 8123888
7946255 1277859
20 499940746
8500064 4362942
9301537 4668807
2755055 9934226
626021 1405039
5130031 1715243
3591332 2974059
6618965 6236119
7816750 1615750
7689419 8709054
8860444 6449613
9444717 5597393
8665043 2853788
3978575 4039427
6471724 3558002
5432693 3431216
3600427 9434455
6530830 9183383
6035856 9063037
6490200 8192055
2379861 7362828
30 507799036
8611630 7246244
6021523 1171386
5165540 4591553
725396 121162
1780222 4791690
2311376 353283
205717 5620763
2687533 878074
8629650 2193220
1455343 8240473
136499 5244354
7198291 8823240
2653749 4308383
4597314 6670528
859820 7454702
1701195 7707803
6935275 5979321
5656636 5433897
6589230 7151610
5154757 2613365
6064770 3919316
701039 1649040
1450215 9036228
3057865 9845769
8631347 2413414
5304083 127665
6502662 1085431
8056067 1334736
7754007 3579085
4426740 5165410
15 833010045
5407536 1727496
6943573 3674706
9652861 7442561
5469377 6316874
993023 3955766
757619 4004103
7137811 7367622
5959821 1408929
1674803 7120652
6280579 4051773
8629611 4676975
7331591 4096187
466336 4891334
3064876 7756187
6986913 4628525
30 109608530
4421701 5773444
9128606 5143304
7916095 7035499
4348087 6234291
1001615 7114541
956587 9147888
5138889 3546776
4668882 7884403
7203879 1197595
2893821 7337917
566439 9237938
759939 7202719
2887064 7851093
9949765 1242190
9734925 3937652
1833346 3443722
9776745 4614713
4107274 9827167
8835831 1425743
4021697 6687718
5998433 8136630
8495718 747329
8442735 9181521
5542908 6952709
5179379 2335782
6184497 4852467
4374845 9704422
5737285 8874814
5189631 2831985
6964705 8321856
13 597985415
2711143 7075740
6880876 4398511
7883845 7321376
733867 3538580
6967737 3295403
8408283 956433
9553857 2156266
2194724 9296067
5740827 6263709
2274192 1189058
5214403 7911895
9182494 9064226
4766524 2294162
11 408228496
8503664 8264923
677401 416272
8675096 9993309
9558853 4388086
4932034 8690443
9596634 5004226
7246083 4885158
9695167 1406079
9252204 1189124
3521696 4023389
9183396 315650
28 487004862
1840804 5915915
2910669 3070278
4453462 3837340
394424 9339295
2906743 4443222
6990705 9422906
8618150 7826836
7994467 2553261
3819228 9777060
7928600 8679518
8466285 2358948
8887569 1484664
4600060 1321129
3272645 6601010
5107226 2136769
4114516 2691336
6695740 6755342
3043979 7440709
4719093 4243346
743666 9957760
8057518 3887134
7658756 6593431
4314565 2490186
4920630 5252242
7026608 5203446
5751459 7791674
6368777 5228416
6779264 2921025
19 968017842
4825010 630709
1180454 6019449
499490 6422247
9702341 1527451
858233 2529205
2951106 9858728
7298075 5348833
8969437 7286616
1860575 1373222
5168607 6148431
8235734 6391293
812545 2430083
1851543 8735037
5533128 8411739
7551382 3658570
5396363 9957285
3498738 3760467
2561356 5951403
939015 5290625
28 40381370
884691 9854495
1895336 4382660
6995562 4382108
4111653 8149696
3920916 6032675
9698055 8075745
9919700 953265
4892776 5436312
6652977 4036346
2529806 7919711
3129223 3815633
7104348 4754124
3171049 2478126
3268065 5833966
1097303 314833
2759832 7999604
6041825 5455072
9419383 9201529
4392612 5142150
3928466 4174723
7568357 1599373
1007531 9302266
7072877 6267051
8496522 6894143
7847086 6957328
786607 6888547
9075705 7311204
3057048 150416
21 548738819
4542926 1441508
9976114 5011143
4905132 8695399
6777441 2098249
163115 5559252
2952678 3226101
6370856 894769
9060669 8989358
1527699 9448853
6492988 1890282
4781558 6127905
4056820 1376225
2758074 3091562
7102633 4532602
6646262 1087637
1533374 7473447
2184867 9796280
1597500 2817543
2866719 5510426
9186953 2206895
8830008 9402195
20 244094048
9013465 2617043
5407297 7161155
5926711 8174235
6980912 4936637
3953464 2010487
5562722 9733009
3353721 7787497
6476176 9817261
8580385 7578112
3929436 2462544
9618057 8770183
4372532 5451228
8658577 691226
6151169 8309229
5643466 4168907
1856970 4070837
8333501 4789050
1628901 6955881
7988102 4248410
7907682 8099494
13 383620907
7428997 4041257
5209000 8073004
5140767 4718521
2307349 3864504
6403050 6620069
1644503 489218
2886112 8728309
2852676 1990425
7020103 4579237
5266917 5398045
3706172 459404
6656712 2995169
6958005 9180725
11 49316900
8759089 7143321
3022853 403268
6625031 3451956
1335277 8996414
3280836 1591998
534628 1372086
6254778 2650175
1614173 8987130
1535982 3334834
2246973 862032
2744109 3453519
12 169116646
1037333 7962684
637542 7784063
3708228 3943799
5487258 5053547
757238 8072033
2839499 2613945
3680933 9339456
9873149 6590275
6620242 6255980
5911410 9005321
4262049 3534772
452087 7461204
16 846265248
1384879 329199
4182477 7989196
2481911 545626
8548103 3164859
6696399 6215754
1147456 6536177
4682785 4865722
9826961 9595527
6225435 2704133
6945398 7207671
721875 628910
4466276 5798030
1259370 1941350
9709101 1902685
6766652 2847277
3803343 9950992
