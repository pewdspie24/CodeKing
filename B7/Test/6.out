261
61
95
29
12
81
76
182
13
80
46
66
119
5
79
31
104
13
50
109
